packages/core/src/render3/interfaces.ts

```diff
@@ -245,8 +253,11 @@ export interface QueryState {
    * @param queryList `QueryList` to update with changes.
    * @param predicate Either `Type` or selector array of [key, value] predicates.
    * @param descend If true the query will recursively apply to the children.
+   * @param read Indicates which token should be red from DI for this query.
```

Comment:
`red` => `read`. I know past tense of `read` is `read` but it is pronounced `red` like the color